miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -111,6 +119,11 @@ abstract class MiniAppMessageBridge {
         this.isAdMobEnabled = isAdMobProvided()
     }
 
+    /** Set implemented userInfoHandler. Can use the default provided class from sdk [UserInfoHandler]. **/
+    fun setUserInfoHandler(handler: UserInfoHandler) {
+        this.userInfoHandler = handler
+    }
```

Comment:
There is a lack of initialization for `setUserInfoHandler` here. The developer may call `setUserInfoHandler` after miniapp webview is created.
Here is [suggestion](https://github.com/khairul-alam-licon/android-miniapp/blob/bb1083fc3263c6878592d64adb69f27b2a140665/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt#L129) from my PR.